bedrock/newsletter/utils.py

```diff
@@ -76,3 +72,22 @@ def custom_unsub_reason(token, reason):
         'reason': reason,
     }
     return basket.request('post', 'custom_unsub_reason', data=data)
+
+
+def get_local_basket_newsletters_data():
+    """
+    Load newsletter data from a file previously saved from basket
+
+    To update the data run the following:
+
+        $ curl https://basket.mozilla.org/news/newsletters/ > bedrock/newsletter/basket_data.json
```

Comment:
Not a blocker for this PR, but I think we're going to need to either automate the updates of the json file in version control or have a documented manual update schedule so that we don't get too far out of sync.